### DISCUSSION TITLE
Airflow Operator uses Config API

### DISCUSSION THREAD

**Discussion by im-voracity** [airbytehq/airbyte#41664] [Platform API Questions] [answered]
I see that the Airbyte team is making efforts to migrate the Configuration API (which is being deprecated) to the Airbyte API.

I use Airflow to orchestrate Airbyte, and the official Airbyte provider (at least for OSS) seems to only use the Configuration API.

When the Configuration API is deprecated, will the Airbyte provider stop working? Are there any plans to migrate the provider to the new Airbyte API?

**Comment by marcosmarxm** [comment]
@im-voracity I'm working to migrate the integrations to use the Python SDK which uses the Airbyte API.

**Reply by im-voracity** [reply]
So the Airbyte provider for Airflow should be updated to use Airbyte API when the Config API is deprecated?

**Comment by marcosmarxm** [comment, marked as answer]
This was fixed in https://github.com/apache/airflow/pull/41122
